packages/core/strapi/lib/middlewares/security.js

```diff
@@ -33,10 +33,12 @@ const defaults = {
 module.exports = config => (ctx, next) => {
   let helmetConfig = defaultsDeep(defaults, config);
 
-  if (
-    ctx.method === 'GET' &&
-    ['/graphql', '/documentation'].some(str => ctx.path.startsWith(str))
-  ) {
+  const PLUGIN_ENDPOINTS = [
+    strapi.plugin('graphql').config('endpoint'),
```

Comment:
you have to check plugins exist in the project before using them. YOu will see the change if you pull the main branch :)

Reply:
Can we assume the documentation plugin will always be present?

Reply:
Nope :) It is not even installed by default actually